### PR TITLE
ENH: more didactic deduplicate example

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
- Add the functions to the generated docs
- Incorporate the random number generation seed accordingly to scikit-learn's standard
- A bit of phrasing

### CODE REVIEW

File: examples/06_deduplication.py [hunk 1]
```diff
@@ -90,6 +85,12 @@
 # a lighter color means a larger distance. We can see that we are dealing with three
 # clusters - the original medication names and their misspellings that cluster around them.
 
+from dirty_cat._deduplicate import compute_ngram_distance
```
Comment (jovan-stojanovic) [inline examples/06_deduplication.py:88]:
```suggestion
from dirty_cat import compute_ngram_distance
```
This function was actually made public and added to the init, so there is no need to import it from a private submodule #466

**Review by jovan-stojanovic — CHANGES_REQUESTED** [review]
Simple solution for the first point in #466

**Comment by GaelVaroquaux** [comment]
I applied the suggested change and resolved conflicts. Thanks!

**Comment by jovan-stojanovic** [comment]
Merged, thanks!

**Pull request merged**